koku/masu/openapi.json

```diff
@@ -847,10 +856,29 @@
           "type": "object",
           "properties": {
             "database": {
-              "type": "string"
+              "type": "string",
+              "nullable": "true"
             },
             "database_connections": {
-              "type": "string"
+              "type": "integer",
+              "format": "int64"
+            }
+          }
+        },
+        "CeleryResponse": {
```

Comment:
This is the response content that is actually missing from https://issues.redhat.com/browse/COST-1174 but like I mentioned, I'm not sure its accessible without having it as a new endpoint rather than a param on /status/ . I could be wrong but if not then we would need to change masu to add this endpoint for this to work.